integration-tests/tests/integration/snapshots/integration__archive_unsupported_build_filter.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: integration-tests/tests/integration/main.rs
+expression: output.stderr_as_str()
+---
+  error: predicate not allowed in `test-archive-filter` expressions
+   ╭────
+ 1 │ test(sample_test)
+   ·      ─────┬─────
+   ·           ╰── this predicate causes an unsupported expression
```

Comment:
Could you reword this as "test predicates are not supported while archiving"? You may have to update the other messages as well instead of saying "this predicate causes {}".

Reply:
Tweaked the wording a bit